### ISSUE TITLE
allow docs to be modified, ala wiki

### ISSUE THREAD

**Issue by Morriz** [strongloop/loopback#2134]
There are many errors and omissions in the docs, so I propose you allow the community to make changes for moderation.

**Comment by crandmck** [comment]
I would like to do this as well, because I simply cannot handle all the work that needs to be done.  However, it's a bit complex because we're using a Confluence wiki and reusing the content as part of a commercial product, API Connect.  

Ultimately, I want to extract all the docs that pertain just to the LoopBack framework (and not the tools `slc` and Arc, and their equivalents in API Connect) and move those docs into a more-easily edited format, e.g. Markdown/Jekyll, which we're using for http://expressjs.com.  But that migration will be a large project itself.

In the meantime, I'll try opening editing of the LoopBack docs to anonymous users.  When we tried that previously, we got a bunch of spam, so we had to lock it down.  When I have a moment, I will try it again, and I'll have to write up instructions on how to edit as well.
